docs/modules/ROOT/pages/index.adoc

```diff
@@ -408,12 +433,33 @@ The validator SHOULD verify that no entry in `signer_payload.referenced_assertio
 
 The validator MUST ensure that `signer_payload.referenced_assertions` contains at least one _<<_hard_binding,hard binding>>_ assertion as described in link:++https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html#_hard_bindings++[Section 9.2, “Hard bindings”] of the C2PA technical specification. The `cawg.identity.hard_binding_missing` error code SHALL be used to report a missing _<<_hard_binding,hard binding>>_ assertion.
 
-The validator MUST maintain a list of valid `sig_type` values and corresponding code paths for the `signature` values that it is prepared to accept. Validators SHOULD be prepared to accept all signature types described in xref:_credentials_signatures_and_validation_methods[xrefstyle=full]. The `cawg.identity.sig_type.unknown` error code SHALL be used to report assertions that contain unrecognized `sig_type` values.
+The validator MUST maintain a list of valid `signer_payload.sig_type` values and corresponding code paths for the `signature` values that it is prepared to accept. Validators SHOULD be prepared to accept all signature types described in xref:_credentials_signatures_and_validation_methods[xrefstyle=full]. The `cawg.identity.sig_type.unknown` error code SHALL be used to report assertions that contain unrecognized `signer_payload.sig_type` values.
 
 The `signature` field of an *<<_identity_assertion,identity assertion>>* MUST contain a valid signature. The procedure for validating each signature type and corresponding status codes are described in xref:_credentials_signatures_and_validation_methods[xrefstyle=full].
 
 The `pad1` and `pad2` fields of an *<<_identity_assertion,identity assertion>>* MUST contain only zero-value (`0x00`) bytes. The `cawg.identity.pad.invalid` error code SHALL be used to report assertions that contain other values in these fields.
 
+==== Validating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>>* contains an `expected_claim` value in `signer_payload`, it MUST be validated as follows:
+
+. Locate the _<<C2PA claim>>_ data structure that includes this *<<_identity_assertion,identity assertion>>.*
+. Replace the `hash` value for the specific *<<_identity_assertion,identity assertion>>* being validated with 32 zero-value (`0x00`) bytes.
```

Comment:
I don't think that this works if there is more than ID assertion (or, at least, it will only work for the last ID assertion that is created)

Reply:
I see that this is addressed earlier in your PR.  Perhaps the earlier text should say something like:

"The expected-claim value can only be calculated when all other assertion data has been finalized.  In the context of this specification, this means that only the ID assertion generated last can use 'expected-claim'.

Reply:
Since we have a dependency on the C2PA claim structure, do we need to specify the C2PA version used for computing the hash in case the structure changes over time within the C2PA spec?  You could argue that you don't since the verifier will use whatever is in front of them regardless of CAWG/C2PA versions. However, I thought that I would raise the question.